src/gp_engine.cpp

```diff
@@ -41,7 +43,8 @@ void GPEngine::operator()(const GPOperations::Zero& op) {
 void GPEngine::operator()(const GPOperations::SetToStationaryDistribution& op) {
   auto& plv = plvs_.at(op.dest_);
   for (size_t row_idx = 0; row_idx < plv.rows(); ++row_idx) {
-    plv.row(row_idx).array() = stationary_distribution_(row_idx);
+    plv.row(row_idx).array() =
```

Comment:
I think I asked for a comment here about why this makes sense.

Reply:
I initially put it in the header file. It's now moved here.